### ISSUE TITLE
Deprecate this repository

### ISSUE THREAD

**Issue by ErinBecker** [carpentries/git-novice-es-archive#23]
This repository has been imported to the swcarpentry organization: https://github.com/swcarpentry/git-novice-es

The website can be viewed here: https://swcarpentry.github.io/git-novice-es/

We are organizing a review of the text prior to publication of this lesson on the SWC lessons page and on Zenodo.

@orchid00 - could you please close this repository?

**Comment by orchid00** [comment]
Hi @ErinBecker 
does close a repository mean to delete the repo ?
Are we going to directly make PRs in the new repo?

**Comment by ErinBecker** [comment]
Yes, we can make changes directly to the new repo. In the past, it has
sometimes been confusing for contributors to have multiple forks in
different orgs and closing this repo would prevent that confusion. Rather
than deleting, however, we can archive the repo. I'm not sure how but I
know that GitHub allows this.
